Makefile

```diff
@@ -1,16 +1,17 @@
-all: build test sanity
-
-.PHONY: build
-build:
-	cargo build --release
+.PHONY: linux
+linux:
+	cargo build --release --target x86_64-unknown-linux-gnu
 	cargo install --path .
 	cargo clippy
 	cargo fmt
-
-.PHONY: test
-test:
 	cargo test --release
-
-.PHONY: sanity
-sanity:
 	python3 scripts/sanity_check.py
+
+.PHONY: macos
+macos:
+	rustup target add aarch64-apple-darwin
+	cargo build --release --target=x86_64-apple-darwin
+	cargo build --release --target=aarch64-apple-darwin
+	cargo install --path .
```

Comment:
must run `cargo test --release`

Reply:
now that i think of it, maybe we should also just run the sanity_script. Seems like  a safe thing to do. It's not that slow. I know I said we don't need to, but maybe it's good to do it.

Reply:
sure we can add it, does the integration and unit tests should work ok on macOS ?

Reply:
yes!